pages/documentation/settings.md

```diff
@@ -19,17 +21,37 @@ subnav:
   href: '#utilities-settings'
 ---
 
-A good way to get started with USWDS settings is to copy the `dist/theme` directory out of the `uswds` package and into the same directory where you keep your project Sass files. Then use the `@forward` directive in Sass to include these settings **before including the main USWDS Sass**. You can use the `styles.scss` example file in the `dist/theme` directory as a starting point for your own Sass entry point:
+## Introducing USWDS settings
+USWDS is built using a suite of customizable settings that allows you to configure the design system to fit your project's needs. These settings are Sass variables that begin with the `$theme-` prefix and are typically defined with [USWDS design tokens]({{ site.baseurl }}/design-tokens). The full list of USWDS settings and their default values can be found in the [tables below](#general-settings).
+
+## Configuring Custom USWDS settings
+All USWDS settings are defined in the `uswds-core` package and can be configured using Sass' [`@use..with()`](https://sass-lang.com/documentation/at-rules/use#configuration) rule. To create a custom configuration of USWDS, add USWDS settings variables with custom values inside the parentheses of this statement, as shown in this example:
 
 ```scss
-@forward 'uswds-theme';
-@forward 'uswds';
-@forward 'uswds-theme-custom-styles';
+@use "uswds-core" with (
+  $theme-show-compile-warnings: false,
+  $theme-show-notifications: false,
+  $theme-image-path: "../uswds/img",
+  $theme-show-compile-warnings: true,
+  $theme-banner-background-color: "ink",
+  $theme-banner-link-color: "primary-light",
+  $theme-input-tile-border-color: "base",
+);
 ```
+Note that the settings variables in this module inform both general and component customizations.
+
+### What to include in your configuration
+Include only USWDS settings that you wish to modify in this configuration. These items should be comma-separated and given values of the appropriate type.
```

Comment:
**Optional**, but what is the appropriate type? Maybe specify that it's usually design tokens again.

Reply:
@bonnieAcameron @mejiaj My word brain is moving slow today. Does this add clarity? 

```suggestion
Include only USWDS settings that you wish to modify in this configuration. These items should be comma-separated and given values of the appropriate type, as identified in the tables below.
```